src/Foundation/Account/code/App_Config/Include/Foundation/Wooli/Foundation.Account.config

```
@@ -4,7 +4,7 @@
   <sitecore>
     <pipelines>
       <wooli.foundation.account.login>
-        <processor type="Wooli.Foundation.Account.Infrastructure.Pipelines.Login, Wooli.Foundation.Account" resolve="true" />
+        <processor type="Wooli.Foundation.Account.Infrastructure.Pipelines.LoginProcessor, Wooli.Foundation.Account" resolve="true" />
```

Here part of namespace is missing
```suggestion
        <processor type="Wooli.Foundation.Account.Infrastructure.Pipelines.Login.LoginProcessor, Wooli.Foundation.Account" resolve="true" />
```